packages/webcert/src/feature/careProvider/CareProviderModal.test.tsx

```diff
@@ -140,5 +140,13 @@ describe('Care provider modal', () => {
       renderComponent()
       expect(screen.getByText('Välj vårdenhet')).toBeInTheDocument()
     })
+
+    it('should not allow closing the modal if choose unit resource link exists', () => {
+      testStore.dispatch(updateUserResourceLinks(getChooseUnitResourceLink()))
+
+      renderComponent()
+      userEvent.click(document.body)
+      expect(testStore.getState().ui.uiUser.isCareProviderModalOpen).toEqual(true)
```

Comment:
This test is closely linked to the implementation of the component. If the isCareProviderModalOpen would be refactored and removed then the test would not make sense. Why don't you test that the modal is open by checking if the modal title exists for example?